File: examples/access-levels-list/access-levels-list.py
import json
import iam_floyd as statement

s = (
    # doc-start
    statement.Ec2() \
    .allow() \
    .all_list_actions()
    # doc-end
).to_json()

pretty = json.dumps(s, indent=4, sort_keys=True)
print(pretty)

File: examples/access-levels-list/access-levels-list.ts
import { deploy, out } from '../../helper/typescript/typescript_test';
import { Statement } from '../../lib';

function getStatement() {
  return (
    // doc-start
    new Statement.S3() //
      .allow()
      .allListActions()
    // doc-end
  );
}
const s = [getStatement()];
out(s);
deploy(s);
